test/inductor/test_kernel_benchmark.py

```diff
@@ -51,9 +52,18 @@ def verify_compiled_kernels(self):
         # make sure we have the bandwidth information in the output
         FileCheck().check_count(
             "GB/s",
-            1,
+            GB_count,
             exactly=1,
         ).run(bench_out)
+        # make sure we correctly generate the grid info
+        if check_grid:
+            with open(compiled_module.__file__) as f:
+                source_code = f.read()
+            FileCheck().check_count(
+                "grid=(608, 1, 1)",
```

Comment:
I bet this test is potentially quite flaky since the grid size depends on the config we pick for triton matmul template which is decided by the autotuning result.

We probably can either read back the block size from the generated wrapper code and use that to compute the grid, or mock the config list to a single config so we know for sure what grid size should be.

Reply:
Yeah, agreed. Fixed. Thanks.